### DISCUSSION TITLE
Very small "drag-to-snap zones" (not sure what to call them)

### DISCUSSION THREAD

**Discussion by SgtSweetieBelle** [rxhanson/Rectangle#1394] [General]
For some reason the "drag-to-snap zones" have become very small when trying to drag a window to the edge of a screen. I feel like I have to be pixel perfect, and my mouse frequently crosses over to another screen instead of engaging the snap behavior.

The problem seems worse at larger resolutions, or on edges leading to another screen. 

I see no options for increasing the size of the snap zone.

macOS version: Sonoma 14.4.1
Rectangle version:

**Comment by SgtSweetieBelle** [comment]
Rectangle version: 0.77 (83)

**Comment by rxhanson** [comment]
Here is the terminal command configuration for the snap area margins:
https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#changing-the-margin-for-the-snap-areas

**Reply by ethaniel** [reply]
If you are on rectangle pro, don't forget to replace "com.knollsoft.Rectangle" with "com.knollsoft.Hookshot". 
But it would be amazing if this feature would be avaialbe in the GUI.
